src/hooks/useDrawPath.tsx

```diff
@@ -11,8 +11,17 @@ export interface UseDrawPathOptions
   extends ControllerHookOptions,
     DualAxisOptions,
     PathOptions {
+  /**
+   * close the path (link the first and last points)
```

Comment:
Start all comments with uppercase
```suggestion
   * Close the path (link the first and last points)
```